bang/models.py

```diff
@@ -692,6 +692,28 @@ def cached_chibis(self):
             'http_image_url': get_http_image_url_from_path(path),
         }) for id, path in zip(split_data(self._cache_chibis_ids), split_data(self._cache_chibis_paths))]
 
+    cameo_members = models.ManyToManyField(Member, related_name='cameo_members', verbose_name=_('Cameos'))
+
+    _cached_cameos_collection_name = 'member'
+    _cache_cameos_days = 200
+    _cache_cameos_last_update = models.DateTimeField(null=True)
+    _cache_j_cameos = models.TextField(null=True)
+
+    def to_cache_cameos(self):
+        them = self.cameo_members.all()
+
+        return [{
+            'id': cameo.id,
+            'name': unicode(cameo.name),
+            'japanese_name': unicode(cameo.japanese_name),
+            'image': unicode(cameo.square_image),
+        } for cameo in them]
```

Comment:
Why create a variable and not have just `} for cameo in self.cameo_members.all()`? For clarity?